### PR TITLE
SCMOD-6281: Apollo BackEnd Services (Gamma): Fixed Fortify version mismatches

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://portal.digitalsafe.net/browse/SCMOD-6281

http://sou-jenkins2.hpeswlab.net/job/JobService/view/Developer/job/JobService~job-service~SCMOD-6281~CI/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -298,6 +338,31 @@
...
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-core</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-expression</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
```
Comment (dermot-hardy) [inline pom.xml:365]: Doesn't the worker framework use the spring framework bom?

Comment (thilagasanthosh) [reply pom.xml:365]: Yes Dermot. The worker-framework branch SCMOD-6281 has the changes for spring upgrade. But the build is failing if we use the lastest worker framework.
New Jira created to fix build issue while upgrading latest worker-framework.

https://portal.digitalsafe.net/browse/SCMOD-7023

Comment (dermot-hardy) [reply pom.xml:365]: OK but I want these changes reviewed and simplified as part of SCMOD-7023.  Add a comment to that effect to the description of the Jira.

**Review by dermot-hardy — APPROVED** [review]

**Comment by thilagasanthosh** [comment]
The story SCMOD-7023 is updated for this change.

**Pull request merged**